modules/hub/scripts/gke_hub_registration.sh

```diff
@@ -30,6 +30,7 @@ PROJECT_ID=$5
 tmp_file=$(mktemp)
 # shellcheck disable=SC2064
 trap "rm -rf $tmp_file" EXIT
-echo "${SERVICE_ACCOUNT_KEY}" | base64 --decode > "$tmp_file"
+base64 --help | grep "\--decode" && B64_ARG="--decode" || B64_ARG="-d"
```

Comment:
@cloud-pharaoh base64 should accept both `-d` and ` --decode` right?

Reply:
Depending on OS.
On my mac it does. On my alpine based runner it only supports -d so it broke when I tried to use it.

I saw this pattern in kubectl_wrapper.sh script for multiple modules and it works now :)

Reply:
@cloud-pharaoh makes sense, could you rebase this. For some reason the github UI does not show the update branch button for me.

Reply:
Done 👍